Snakefile

```diff
@@ -4,9 +4,12 @@ examples = [
     'more_renewables_less_fossil'
 ]
 
-scenarios = ["toy-scenario", "toy-scenario-2"]
+scenario_list_example = ['examples']
```

Comment:
With this change, snakemake does not find the variable scenarios any more, which is needed below.